src/main/java/com/denys/toukdemo/controller/ReservationController.java

```diff
@@ -0,0 +1,80 @@
+package com.denys.toukdemo.controller;
+
+import com.denys.toukdemo.dto.Reservation;
+import com.denys.toukdemo.dto.ReservationUnit;
+import com.denys.toukdemo.dto.Room;
+import com.denys.toukdemo.dto.User;
+import com.denys.toukdemo.helper.RoomHelper;
+import com.denys.toukdemo.service.MailService;
+import com.denys.toukdemo.service.ReservationService;
+import com.denys.toukdemo.validation.UserValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@RestController
+public class ReservationController {
+
+    private static final Integer VOUCHER_DISCOUNT_GRADE = 2;
+
+    @Autowired
+    ReservationService reservationService;
```

Comment:
dlaczego to pole nie jest ukryte w klasie? kto potrzebuje się dostawać do niego spoza tej klasy?

Reply:
> dlaczego to pole nie jest ukryte w klasie? kto potrzebuje się dostawać do niego spoza tej klasy?

mój błąd